docs/bnb-smart-chain/developers/quick-guide.md

```diff
@@ -13,6 +13,7 @@ Since BSC is EVM-compatible, your existing Ethereum smart contract skills will s
 ## Connecting
 
 Here are some resources to help you get connected to the BSC network:
+
 - [Network Information and RPC Providers](./rpc.md)
```

Comment:
<img width="763" alt="image" src="https://github.com/user-attachments/assets/d6a15add-1bd1-44ac-bd4d-54c242ee86ca">
not expected